design/yugabyte/yugabytedb-rook-design.md

```diff
@@ -0,0 +1,89 @@
+YugaByte DB is a high-performance distributed transactional database. (More information [here](https://docs.yugabyte.com/latest/introduction/)). This issue outlines design and interfaces for a Rook YugaByte operator for running YugaByte on Kubernetes with Rook as the underlying storage engine.
```

Comment:
I'm seeing 2 files in this PR that appear to be identical:
* `design/yugabyte/yugabytedb-rook-design.md`
* `design/yugabytedb-rook-design.md`

Can we keep just one of those? The file that's in the `yugabyte` subfolder seems good to me.